cmd/geth/chaincmd.go

```diff
@@ -80,28 +80,28 @@ The dumpgenesis command dumps the genesis block configuration in JSON format to
 		Usage:     "Import a blockchain file",
 		ArgsUsage: "<filename> (<filename 2> ... <filename N>) ",
 		Flags: []cli.Flag{
-			utils.DataDirFlag,
-			utils.CacheFlag,
-			utils.SyncModeFlag,
-			utils.GCModeFlag,
-			utils.SnapshotFlag,
-			utils.CacheDatabaseFlag,
-			utils.CacheGCFlag,
-			utils.MetricsEnabledFlag,
-			utils.MetricsEnabledExpensiveFlag,
-			utils.MetricsHTTPFlag,
-			utils.MetricsPortFlag,
-			utils.MetricsEnableInfluxDBFlag,
-			utils.MetricsEnableInfluxDBV2Flag,
-			utils.MetricsInfluxDBEndpointFlag,
-			utils.MetricsInfluxDBDatabaseFlag,
-			utils.MetricsInfluxDBUsernameFlag,
-			utils.MetricsInfluxDBPasswordFlag,
-			utils.MetricsInfluxDBTagsFlag,
-			utils.MetricsInfluxDBTokenFlag,
-			utils.MetricsInfluxDBBucketFlag,
-			utils.MetricsInfluxDBOrganizationFlag,
-			utils.TxLookupLimitFlag,
+			&utils.DataDirFlag,
+			&utils.CacheFlag,
```

Comment:
As I [said before](https://github.com/ethereum/go-ethereum/pull/24751#issuecomment-1110757409), it would be better to make this change in a different way: define the variable as a pointer and leave the references as-is.

Reply:
@fjl that's what I'm doing. This change must have been some git rebase gone wrong, will fix it.